localisation/00_zuwc_united_localisation.csv

```diff
@@ -1442,7 +1442,7 @@ has_3_4_physical_class_trigger_tooltip;Physical class level is 3 or 4\n;Classe p
 has_5_6_physical_class_trigger_tooltip;Physical class level is 5 or 6\n;Classe physique de niveau 5 ou 6\n;Phyische Klassenstufe betr�gt 5 oder 6\n;;;;;;;;;;;x
 has_7_8_physical_class_trigger_tooltip;Physical class level is 7 or 8\n;Classe physique de niveau 7 ou 8\n;Phyische Klassenstufe betr�gt 7 oder 8\n;;;;;;;;;;;x
 has_9_10_physical_class_trigger_tooltip;Physical class level is 9 or 10\n;Classe physique de niveau 9 ou 10\n;Phyische Klassenstufe betr�gt 9 oder 10\n;;;;;;;;;;;x
-EVTDESC_WCCLS_105;You fought in the heat of battle alongside your soldiers. When the first unfortunate enemy soldier died by your hand, his blood splashed onto your tunic. You quickly rushed to the next opponent with a deafening cry!;Vous avez combattu dans le feu de l'action aux c�t�s de vos soldats. Lorsque le premier malheureux soldat ennemi est mort par votre main, son sang a �clabouss� votre tunique. Vous vous �tes rapidement pr�cipit� vers l'adversaire suivant avec un cri assourdissant !;Ihr habt in der Hitze der Schlacht mit Euren Soldaten gek�mpft. Als der erste ungl�ckselige feindliche Soldat durch Eure Hand starb, spritzte sein Blut auf Euren Waffenrock. Ihr seid mit einem ohrenbet�ubenden Schrei blitzschnell auf den n�chsten Gegner zu gest�rmt!;;;;;;;;;;;x
+EVTDESC_WCCLS_105;[target_class.GetFirstName] fights in the heat of battle alongside [target_class.GetHerHis] soldiers. When the first unfortunate enemy soldier dies by [target_class.GetHerHis] hand, his blood splashes onto the tunic. [target_class.GetFirstName] quickly rushes to the next opponent with a deafening cry!;Vous avez combattu dans le feu de l'action aux c�t�s de vos soldats. Lorsque le premier malheureux soldat ennemi est mort par votre main, son sang a �clabouss� votre tunique. Vous vous �tes rapidement pr�cipit� vers l'adversaire suivant avec un cri assourdissant !;Ihr habt in der Hitze der Schlacht mit Euren Soldaten gek�mpft. Als der erste ungl�ckselige feindliche Soldat durch Eure Hand starb, spritzte sein Blut auf Euren Waffenrock. Ihr seid mit einem ohrenbet�ubenden Schrei blitzschnell auf den n�chsten Gegner zu gest�rmt!;;;;;;;;;;;x
```

Comment:
I don't get why you want to change that to speak of the character in 3rd person. It's an event about you yourself fighting, isn't it?

Reply:
I agree with Arithon, I like the first one more.

Reply:
I agree with Arithon, I like the first one more.

Reply:
Changed in 194073d15e32b18f2c3edd36416b2e4a68072ae4.